Templates/HttpPOSTQueueTrigger(CRUD)-FSharp/metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+  "defaultFunctionName": "HttpPOSTQueueTrigger(CRUD)-FSharp",
+  "description": "$HttpPOSTQueueTrigger(CRUD)-FSharp_description",
```

Comment:
Add string similar to [this](https://github.com/Azure/azure-webjobs-sdk-templates/blob/dev/Resources/Resources.resx#L912) in [resource file] (https://github.com/Azure/azure-webjobs-sdk-templates/blob/dev/Resources/Resources.resx) for $HttpPOSTQueueTrigger(CRUD)-FSharp_description